apps/cli/commands/extract_pending.py

```
@@ -0,0 +1,222 @@
+
+        Args:
+            doc_id: Document UUID.
+
+        Returns:
+            str: Document text content.
+
+        Raises:
+            KeyError: If document not found.
+        """
+        if doc_id not in self._content:
+            raise KeyError(f"Document {doc_id} not found")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)